vembrane/__init__.py

```diff
@@ -37,12 +37,44 @@
 }
 
 
+class Map(dict):
+    def __init__(self, *args, **kwargs):
+        super(Map, self).__init__(*args, **kwargs)
+        for arg in args:
+            if isinstance(arg, dict):
+                for k, v in arg.iteritems():
+                    self[k] = v
+        if kwargs:
+            for k, v in kwargs.iteritems():
+                self[k] = v
```

Comment:
can't you just do e.g. `self.__dict__ = kwargs`?

Reply:
sorry, the whole Map thing was a first try to just inherit from one class. But then I realized, the easiest way is just adding the 3 functions. I deleted the Map class